client/src/panels/panel_declarations/InfographicPanel.js

```diff
@@ -174,16 +174,21 @@ Col.propTypes = {
   isGraph: PropTypes.bool,
 };
 
+const flatten_and_filter_children = (children) => _.chain(children)
+  .thru( (children) => _.isArray(children) ? children : [children] )
+  .flatMap( child => {
+    if ( child.type && child.type === Fragment ){
+      return child.props.children;
+    } else {
+      return child;
+    }
+  })
+  .filter( child => child && child.type )
+  .value();
```

Comment:
This was an unrelated diversion, ran in to this really frustrating logic bug in the `StdPanel` component while trying out different layouts for the tp_by_region panel.

Reply:
Logic makes sense
👍